typings/index.d.ts

```diff
@@ -2510,9 +2517,9 @@ declare module 'discord.js' {
     banner?: Base64Resolvable;
   }
 
-  interface GuildEmbedData {
+  interface GuildWidgetData {
     enabled: boolean;
-    channel: GuildChannelResolvable | null;
+    channel?: GuildChannelResolvable;
```

Comment:
Why is this marked as undefined? It can still be set to null (to mark the widget shouldn't generate an invite)

Reply:
oops